cli/ballerina-launcher/src/main/resources/cli-help/ballerina-test.help

```diff
@@ -28,6 +28,12 @@ DESCRIPTION
 
 OPTIONS
 
+     --coverage
+         This is to create coverage reports for the project / package.
```

Comment:
Since we are moving away from the concept of 'package' shall we rename it to 'module'?